deploy/deploy.yaml

```diff
@@ -491,22 +505,12 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
-      - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy@sha256:e10d1d982dd653db74ca87a1d1ad017bc5ef1aeb651bdea089debf16485b080b
-        name: kube-rbac-proxy
-        ports:
-        - containerPort: 8443
-          name: https
       - args:
         - --metrics-addr=127.0.0.1:8080
         - --enable-leader-election
         command:
         - /manager
-        image: quay.io/isolatedcontainers/sandboxed-containers-operator:4.8
+        image: quay.io/openshift_sandboxed_containers/openshift-openshift-sandboxed-containers:4.8.0-1
```

Comment:
It's not entirely clear to me why we have `$(VERSION)`, `4.8.0` and `4.8.0-1` in the same commit. That looks confusing. I believe that having separate commits that explain the rationale each time would be somewhat helpful if someone were to take over and upgrade to 4.9.

Reply:
$(VERSION) is a variable available in the Makefile and I thought it's good to make use of it. Where we don't have it or some other version variable it should be 4.8-latest I think to avoid having to change it everywhere any time we push a new image. Instead we will always tag the latest image with both 4.8.y-n and 4.8.y-latest

Reply:
using $VERSION now in the Makefile and set it to 4.8-latest. everywhere else we're now using 4.8-latest